### PR TITLE
Block account fixes

### PR DESCRIPTION
**Pull request by ArikDay**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-8219

## Description
Fixed conditional tasks with wrong names for Clarizen and Okta integrations.
Added a tasks that retrieves account ids to block from SailPoint IAM.
Minor visibility improvements.
Deprecating Block Account - Generic 

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic_README.md [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
+
 ---
 
-| **Name** | **Description** |  **Required** |
-| --- | --- | --- | 
-| Username | The array of malicious usernames to block. |  Optional |
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| Username | Array of malicious usernames to block. |  | Optional |
+| Tag | PAN-OS Tag name to apply to the username that you want to block. |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic_README.md:35]:
```suggestion
| Tag | PAN-OS tag name to apply to the username that you want to block. |  | Optional |
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic_v2.yml [hunk 2]
```diff
@@ -2315,6 +2326,43 @@ tasks:
...
     isautoswitchedtoquietmode: false
+  "75":
+    id: "75"
+    taskid: 4ebe9bf0-210f-4752-83d0-bb2e2a03262a
+    type: regular
+    task:
+      id: 4ebe9bf0-210f-4752-83d0-bb2e2a03262a
+      version: -1
+      name: Get Account IDs From SailPoint
+      description: Fetch accounts by search/filter parameters (id, display_name, last_refresh, native_identity, last_target_agg, identity_name & application_name) using IdentityIQ SCIM API's.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic_v2.yml:2337]:
```suggestion
      description: Fetch accounts by search/filter parameters (id, display_name, last_refresh, native_identity, last_target_agg, identity_name & application_name) using IdentityIQ SCIM APIs.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic_v2_README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Supported integrations for this playbook:
 * Prisma Cloud IAM
 * Zoom IAM
 * Atlassian IAM
-* GitHub IAM
+* GitHub IAM.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic_v2_README.md:20]:
```suggestion
* GitHub IAM
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_2.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+#### Playbooks
+
+##### Block Account - Generic v2
+
+- Fixed conditional tasks with wrong names for Clarizen and Okta integrations.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_2.md:6]:
```suggestion
- Fixed conditional tasks with incorrect names for Clarizen and Okta integrations.
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_2.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+- Added a tasks that retrieves account ids to block from SailPoint IAM.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_2.md:7]:
```suggestion
- Added a tasks that retrieves account IDs to block from SailPoint IAM.
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_2.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+- Minor visibility improvements
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_2.md:8]:
```suggestion
- Minor visibility improvements.
```

**Comment by ShirleyDenkberg** [comment]
@AdiPeret Doc review completed.

**Review by AdiPeret — APPROVED** [review]

**Pull request merged**